frontend/lib/api/tool_spec.py

```
@@ -0,0 +1,28 @@
+def load_file_content(filename):
+    try:
+      with open(filename, "rt") as f:
+          return {
+              "result": f.read()
+          }
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)